server/src/main/kotlin/org/web3j/openapi/server/ExceptionMappings.kt

```diff
@@ -57,6 +58,7 @@ class TransactionExceptionMapper : BaseExceptionMapper<TransactionException>(Sta
 class UnsupportedOperationExceptionMapper : BaseExceptionMapper<UnsupportedOperationException>(Status.BAD_REQUEST)
 class IllegalArgumentExceptionMapper : BaseExceptionMapper<IllegalArgumentException>(CustomStatus.UNPROCESSABLE_ENTITY)
 class ContractCallExceptionMapper : BaseExceptionMapper<ContractCallException>(CustomStatus.UNPROCESSABLE_ENTITY)
+class NotFoundExceptionMapper : BaseExceptionMapper<NotFoundException>(CustomStatus.UNPROCESSABLE_ENTITY)
```

Comment:
I don't think we need that, Jersey will translate WebApp exceptions to their status code.

In case we need it, the status should be `Status.NOT_FOUND`.